src/middlewared/middlewared/plugins/pool.py

```diff
@@ -685,6 +697,10 @@ def add(attr):
         ('edit', _add_inherit('readonly')),
         ('edit', _add_inherit('recordsize')),
         ('edit', _add_inherit('snapdir')),
+        ('add', Inheritable('quota_warning', value=Float('quota_warning', validators=[Range(0.0, 1.0)]))),
```

Comment:
Why these are only on update and not on create?

Reply:
> Why these are only on update and not on create?

It's to be consistent with other properties. You can't pass `INHERIT` as property value when creating dataset. If you want `INHERIT`, do not specify property at all, it will be default.

Reply:
I dont know how that comment got in, I figured that out before sending the review. Sorry : P